javascript/exercise1/ex1/index.js

```diff
@@ -0,0 +1,5 @@
+const logo = document.querySelector('.lnXdpd');
+logo.setAttribute('srcset', 'https://logos-world.net/wp-content/uploads/2020/10/Yahoo-Logo-700x394.png');
```

Comment:
Should store the url to a variable first as the exercise requirement. Pls read more carefully.